### PR TITLE
Order

### PR DESCRIPTION
**Pull request by Someboty**
Created OrderItem and Order entities with DTOs, mapper, controller, service, repository

### CODE REVIEW

**Review by keda-vlad — APPROVED** [review]
LGTM

File: src/main/java/com/bookshop/config/OpenApiConfig.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class OpenApiConfig {
+    @Bean
+    public OpenAPI customOpenApi() {
```
Comment (vovasalyha) [inline src/main/java/com/bookshop/config/OpenApiConfig.java]: try to avoid adding changes which are not related to the current PR

File: src/main/java/com/bookshop/service/impl/OrderServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    public Set<OrderDto> getHistory(Long userId) {
+        return orderRepository.findAllByUserId(userId).stream()
+                .map(orderMapper::toDto)
+                .collect(Collectors.toSet());
+    }
+
+    @Override
+    @Transactional
+    public OrderDto createOrder(Long userId, ShippingAddressRequestDto request) {
+        ShoppingCart cart = cartRepository.getShoppingCartByUser_Id(userId);
```
Comment (vovasalyha) [inline src/main/java/com/bookshop/service/impl/OrderServiceImpl.java]: can repository return null here?

**Review by vovasalyha — APPROVED** [review]
